src/containers/home/index.tsx

```diff
@@ -159,7 +162,13 @@ const Home: React.FC<HomeContainerProps> = ({ announcements }) => {
           <p>Loading announcements...</p>
         )}
         {asyncRequestIsComplete(announcements) && (
-          <AnnouncementsList announcements={announcements.result} />
+          <AnnouncementsList
+            announcements={
+              announcements.result.length > 3
+                ? announcements.result.slice(0, 3)
```

Comment:
Why was the constant `ANNOUNCEMENTS_LIMIT` removed? It would be easier to interpret (and change) than having a magic number imo